parachains/common/src/lib.rs

```diff
@@ -92,7 +92,7 @@ mod constants {
 	pub const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
 
 	/// We allow for 0.5 seconds of compute with a 6 second average block time.
-	pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND / 2;
+	pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND.scalar_saturating_mul(2);
```

Comment:
Why did you change divided by 2 to multiplied by 2?

Reply:
Until async backing is in, our limit is still half a second AFAIK.

Reply:
yeah bug, sorry